### PR TITLE
monstdat cleanup

### PR DESCRIPTION
**Pull request by kphoenix137**

### CODE REVIEW

**Comment by qndel** [comment]
I wouldn't go that far

**Comment by kphoenix137** [comment]
> I wouldn't go that far

?

File: Source/monstdat.h [hunk 1]
```diff
@@ -222,9 +209,6 @@ enum _monster_id : int16_t {
 	MT_BTBLACK,
 	MT_RBLACK,
 	MT_UNRAV,
-	MT_HOLOWONE,
-	MT_PAINMSTR,
-	MT_REALWEAV,
```
Comment (AJenbo) [inline Source/monstdat.h:227]: Deleting thease will shift the ids which is probably going to break a ton of things?

Comment (kphoenix137) [reply Source/monstdat.h:227]: I tested it out, no crashes or issues as far as I can tell.

Comment (AJenbo) [reply Source/monstdat.h:227]: Did you load a single player savegame?

Comment (kphoenix137) [reply Source/monstdat.h:227]: Just teleported through all crypted, killed na krul. all monsters maintained the correct data and names

Comment (AJenbo) [reply Source/monstdat.h:227]: I would also expect this to break loading some quest levels.

Comment (kphoenix137) [reply Source/monstdat.h:227]: > Did you load a single player savegame?

no, let me test

Comment (kphoenix137) [reply Source/monstdat.h:227]: Just tested, no issues

Comment (kphoenix137) [reply Source/monstdat.h:227]: > I would also expect this to break loading some quest levels.

I will test

Comment (kphoenix137) [reply Source/monstdat.h:227]: No crash in Leoric's Tomb. Should I test more levels?

Comment (qndel) [reply Source/monstdat.h:227]: aren't most of these monsters fully functional? Removing working stuff like this isn't beneficial at all

**Comment by kphoenix137** [comment]
I'm going to create an enum class for MonsterID so this pr won't be needed

**Pull request closed without merging**